app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -197,12 +197,7 @@ class ResponseRedacter
         def self.redact(hash)
           return { error: 'TMx response body was empty' } if hash.nil?
           return { error: 'TMx response body was malformed' } unless hash.is_a? Hash
-          filtered_response_h = hash.slice(*ALLOWED_RESPONSE_FIELDS)
-          unfiltered_keys = hash.keys - filtered_response_h.keys
-          unfiltered_keys.each do |key|
-            filtered_response_h[key] = '[redacted]'
-          end
-          filtered_response_h
+          hash.slice(*ALLOWED_RESPONSE_FIELDS)
```

Comment:
You can read this PR as lines 201-204 being deleted, and then me removing the variable assignment on line 200.